### DISCUSSION TITLE
Enhancer / Primordial Wave / Splintered Elements

### DISCUSSION THREAD

**Discussion by glitch1one** [Hekili/hekili#1473] [Q&A] [answered]
Setup: Necrolord Enhancer, Tumbling Waves Conduit (Chance to reset Primordial Wave CD), Unity Leggy / Splintered Elements

Topic: When you cast Primordial Wave and it does get a Cooldown Reset via the Tumbling Waves Conduit it recommends the first lightning bolt and then recast Primordial Waves followed by Lightning bolt when enough Maelstrom is available.

This seems odd to me since the Splintered Elements buff is 12 seconds long and when you cast another lightning bolt it justs simply refreshes the duration - it may also be that you get a lower haste buff cause maybe some of the targets died off so you hit less enemies with lightning bolt.

am i missing something here or am i observing something wrong?

**Comment by Hekili** [comment, marked as answer]
The short answer is "this is what has been shown to work in sims."

How often are you running into this, where you get the CD reset?  I did a quick sim of the Enhancement Necrolord profile and it only happened once in a 5 minute fight, but it is definitely prone to happening.  Are you seeing it more often?

In the sim, use of Primordial Wave isn't built around whether Splintered Elements is already up or anything like it.

`primordial_wave,if=buff.primordial_wave.down&(raid_event.adds.in>42|raid_event.adds.in<6)`

`lightning_bolt,if=buff.maelstrom_weapon.stack>=5&buff.primordial_wave.up&raid_event.adds.in>buff.primordial_wave.remains&(!buff.splintered_elements.up|fight_remains<=12)`
`lightning_bolt,if=buff.maelstrom_weapon.stack=10&buff.primordial_wave.down`
`lightning_bolt,if=buff.maelstrom_weapon.stack>=5&buff.primordial_wave.down`

If you're seeing PWave > LB > PWave > LB, submit a ticket with a snapshot that shows that Primordial Wave is up, Splintered Elements is up, and Lightning Bolt is recommended.  I will review it.

**Reply by glitch1one** [reply]
> The short answer is "this is what has been shown to work in sims."

**well :D**

> How often are you running into this, where you get the CD reset? I did a quick sim of the Enhancement Necrolord profile and it only happened once in a 5 minute fight, but it is definitely prone to happening. Are you seeing it more often?

**I didn't pay attention much to this. i just observed it a few times.**

> If you're seeing PWave > LB > PWave > LB, submit a ticket with a snapshot that shows that Primordial Wave is up, Splintered Elements is up, and Lightning Bolt is recommended. I will review it.

**alright, ill keep an eye out for this**

Thank you for answering to this. i guess its also quite minor since the splintered elements is maybe not that worth compared to a lightning bolt that hits up to 6 targets
